.ci/onnx/jenkins/Jenkinsfile.groovy

```diff
@@ -174,8 +174,12 @@ def main(String label, String projectName, String projectRoot, String dockerCont
                     RunToxTests(configurationMaps)
                 }
             }
-            catch(hudson.AbortException e) {
-                currentBuild.result = 'ABORTED'
+            catch(e) {
```

Comment:
Let's delete `Jenkinsfile.groovy` altogether, if it's no longer needed.

Reply:
O want to delete it in next PR, to make sure the switch is smooth.

Reply:
OK